compiler/optimizer/LoopReducer.cpp

```diff
@@ -357,18 +357,9 @@ TR_Arraycopy::checkArrayStore(TR::Node * storeNode)
 bool
 TR_ByteToCharArraycopy::checkArrayStore(TR::Node * storeNode)
    {
-   if (storeNode->getOpCodeValue() != TR::cstorei)
```

Comment:
This doesn't look right. I would have thought this should change to `!= TR::sstorei` with appropriate updates within the optimization... As it stands the opt will never do anything now...

Reply:
The mistake is fixed in bd43e4f